compiler/fuzzer/src/fuzzer.rs

```diff
@@ -18,18 +18,18 @@ use std::sync::Arc;
 use tracing::debug;
 
 pub struct Fuzzer {
-    lir: Arc<Lir>,
+    pub lir: Arc<Lir>,
     pub function_heap: Heap,
     pub function: Function,
     pub function_id: Id,
+    pub pool: InputPool,
```

Comment:
Can you write getters instead for the newly `pub` fields so that they cannot be modified from the outside?